marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+{
+    "author_name": "Datadog",
+    "description": "",
+    "layout_type": "free",
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "cluster",
+            "prefix": "host"
```

Comment:
Actually no, we don't tag by cluster, but since users should have one integration instance per cluster, a host tag is equivalent to one cluster.
I agree it can seems weird, so maybe I should remove this template variable to avoid confusion

Reply:
Waiting for #7960